test/cpp_extensions/libtorch_agnostic_extension/libtorch_agnostic/csrc/kernel.cpp

```diff
@@ -278,14 +278,29 @@ void boxed_empty_like(StableIValue* stack, uint64_t num_args, uint64_t num_outpu
   stack[0] = from(res);
 }
 
+Tensor my_fill_(Tensor t) {
```

Comment:
```suggestion
Tensor fill_infinity(Tensor t) {
```

nit: more indicative